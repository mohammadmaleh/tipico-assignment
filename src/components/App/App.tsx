import * as React from "react";
import { Provider } from "react-redux";
import configureStore from "../../redux";
import SearchRepositories from "../../containers/SearchRepositories/SearchRepositories";

const store = configureStore();
const App: React.FC = () => {
  return (
    <Provider store={store} data-test="component-app">
      <SearchRepositories />
    </Provider>
  );
};

export default App;
