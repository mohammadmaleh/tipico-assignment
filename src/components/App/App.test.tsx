import * as React from "react";
import { shallow } from "enzyme";
import App from "./App";
import { findByTestAttr } from "../../../test/testUtils";
const defaultProps = {};

const setup = (props = {}) => {
  const setupProps = { ...props };
  return shallow(<App {...setupProps} />);
};
test("it renders without error", () => {
  const wrapper = setup();
  const component = findByTestAttr(wrapper, "component-app");
  expect(component.length).toBe(1);
  expect(component).toMatchSnapshot();
});
