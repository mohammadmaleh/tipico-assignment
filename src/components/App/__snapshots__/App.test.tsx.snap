// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders without error 1`] = `ShallowWrapper {}`;
